foodsaving/base/templates/_base-head.mjml

```diff
@@ -36,10 +36,6 @@
         color: #4a982e;
     }
 
-    .message-content {
```

Comment:
This possibly changes the formatting of the message notification e-mail in an unwanted way (especially with markdown headlines). Maybe we shouldn't use the same class.